ga-ccd-definition/FixedLists/HearingLengthCasePro-GAR3CP-nonprod.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "ID": "HearingLengthCasePro",
```

Comment:
could we reuse the existing one?

Reply:
Created a new JIRA task for refactoring https://tools.hmcts.net/jira/browse/CIV-6823